src/Sulu/Component/Rest/ListBuilder/Filter/DateTimeFilterType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Rest\ListBuilder\Filter;
+
+use Sulu\Component\Rest\ListBuilder\FieldDescriptorInterface;
+use Sulu\Component\Rest\ListBuilder\ListBuilderInterface;
+
+class DateTimeFilterType implements FilterTypeInterface
+{
+    public function filter(
+        ListBuilderInterface $listBuilder,
+        FieldDescriptorInterface $fieldDescriptor,
+        $options
+    ): void {
+        if (!is_array($options) || !isset($options['from']) || !isset($options['to'])) {
```

Comment:
shouldnt it be possible to only set `from` or `to`?

Reply:
Maybe, didn't implement it for now, in order to get this finished :see_no_evil:

Reply:
okay, shouldnt be hard to implement this later on